security/voters.rst

```diff
@@ -163,12 +170,14 @@ would look like this::
             return true;
         }
 
-        protected function voteOnAttribute(string $attribute, mixed $subject, TokenInterface $token): bool
+        protected function voteOnAttribute(string $attribute, mixed $subject, TokenInterface $token, ?Vote $vote = null): bool
         {
             $user = $token->getUser();
+            $vote ??= new Vote();
```

Comment:
should be removed while accounting for @javiereguiluz's comment:
```suggestion
```

Reply:
Fixed while merging. Thanks.